README.md

```diff
@@ -10,15 +10,24 @@ applied and modified to color tabs by project root folder and optionally
 by subfolder levels within the project tree.
 
 ## Edit per project rules
-Use menu `Packages -> Color Tabs by Project -> Edit Rules` to open the rules editor:
+There are two ways of setting custom rules for your projects.
+Use menu `Packages -> Color Tabs by Project -> Edit Rules` to open the rules editor, then:
++ if `Custom rules based on` is set to `project path`
 ```cson
 projects:
   "/your-project-path/":
     folderDepth: 1
```

Comment:
I think we can leave folderDepth out of the examples, as this defaults to 0 and I'm planning to deprecate that feature.